app/models/country.rb

```diff
@@ -1,3 +1,7 @@
 class Country < ApplicationRecord
   has_many :organisations
+
+  scope :with_organisations, -> {
+    joins(:organisations).where('organisations.country_id IS NOT NULL').uniq
```

Comment:
not sure we need the where clause here - `joins` will perform an inner join, so those records will be eliminated automatically. But it would make sense to filter out SYSTEM_MANAGERS organisations - those will always have access anyway.